maps/mjolnir/bos/structures/statue.dm

```diff
@@ -1,41 +1,41 @@
 /obj/structure/memstatue
-	name = "statue stand"
-	desc = "You will get here one day. Or it's just a statue honouring some piece of tile."
-	icon = 'maps/torch/bos/icons/obj/structures/statue.dmi'
-	icon_state = "placeholder"
-	anchored = TRUE
+    name = "statue stand"
+    desc = "You will get here one day. Or it's just a statue honouring some piece of tile."
+    icon = 'maps/torch/bos/icons/obj/structures/statue.dmi'
+    icon_state = "placeholder"
+    anchored = TRUE
 
 /obj/structure/memstatue/roger
-	name = "Roger Blackmore statue"
-	desc = "A native of Tersten, a war hero who was head of the Tersten cell of the League of Struggle. Led the takeover of Tersten City, marched in the forefront of the rebels, blew himself up next to one of the army's military installations, posthumously a hero of the Sovereign Lordania."
-	icon_state = "shahid"
+    name = "Roger Blackmore statue"
+    desc = "A native of Tersten, a war hero who was head of the Tersten cell of the League of Struggle. Led the takeover of Tersten City, marched in the forefront of the rebels, blew himself up next to one of the army's military installations, posthumously a hero of the Sovereign Lordania."
+    icon_state = "shahid"
 
 /obj/structure/memstatue/matt
-	name = "Matt Striker statue"
-	desc = "Personally killed over ninety SCG commanders during the conflict. Passed away after numerous attempts on his life due to stress."
-	icon_state = "suecidnik"
-
+    name = "Matt Striker statue"
+    desc = "Personally killed over ninety SCG commanders during the conflict. Passed away after numerous attempts on his life due to stress."
```

Comment:
`Passed away after numerous attempts` - > ``Passed away after numerous assassination attempts` since usual 'attempts' without specification can lead to some weird jokes that we don't wanna hear about